keyboards/handwired/pteron/keymaps/default/config.h

```diff
@@ -0,0 +1,4 @@
+#pragma once
+
+/* COL2ROW or ROW2COL */
+#define DIODE_DIRECTION ROW2COL
```

Comment:
This should go in the keyboard config, not here.

Reply:
Besides adding a new keymap I am also making the diode direction configurable for each keymap. that is why I am adding DIODE_DIRECTION to each of the keymaps.

Is the default keymap special in some way that I shouldn't add the DIODE_DIRECTION override here?

Reply:
It's less of the keymap, and more of ... if it's compiled on the configurator, it won't have this set, at all. 

If you need this set, then undefining it, and then defining the correct value in the keymap folder would be the correct way to handle this.

Reply:
got it, done!